programs/uxd/src/instructions/credix_lp/edit_credix_lp_depository.rs

```diff
@@ -21,7 +21,6 @@ pub struct EditCredixLpDepository<'info> {
         mut,
         seeds = [CONTROLLER_NAMESPACE],
         bump = controller.load()?.bump,
-        constraint = controller.load()?.registered_credix_lp_depositories.contains(&depository.key()) @UxdError::InvalidDepository,
```

Comment:
the edit IXs are permissioned, and I see no reason not to be able to edit a non-registered depository. It might be useful to be able to configure a depository before we mark it as usable by the router.